### DISCUSSION TITLE
[M-Z] (20 Gennaio 2020) Esercizio 4

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#454] [Esami & Esoneri M-Z]
## Traccia

<img width="632" alt="image" src="https://github.com/user-attachments/assets/a0605db9-9d6b-4698-8281-046eb83863d8">

**Comment by rimaout** [comment]
<img width="800" alt="image" src="https://github.com/user-attachments/assets/f4d02ae7-951c-4be1-a540-dbd9be1b7781">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20200120.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20200120.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/f28e4fdc-5e72-47bc-b89c-ab6e6072eb28)
